client/src/shared/navigation/index.jsx

```diff
@@ -7,19 +8,27 @@ import MobileNavBar from './mobile';
 import DesktopNavBar from './desktop';
 import { withGetScreen } from 'react-getscreen';
 import { useDispatch } from 'react-redux';
+import { gaPageView } from 'utils';
 
-const Navigation = (props) => {
+const Navigation = ({isMobile, isTablet}) => {
 	const dispatch = useDispatch();
 	const { isAuthenticated, user } = UserSelectors();
 	
 	const logoutHandler = async () => {
 		const response = await logout();
 		if (response.status === 'OK') dispatch(notAuthenticated());
 	};
+
+	const location = useLocation();
+
+	useEffect(() => {
+		// tag page view
+		gaPageView(location, isAuthenticated);
+	}, [location, isAuthenticated]);
 	
 	return (
 		<React.Fragment>
-			{props.isMobile() ||props.isTablet() ?
+			{isMobile() || isTablet() ?
```

Comment:
isMobile is always defined?, what is this condition refer to? why not return true / false rather than a value and undefined?

Reply:
yes, isMobile and isTablet are provided by the withGetScreen HOC which wraps this component. i only added props destructuring here, didn't change anything else regarding these props